<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Log</class>
 <widget class="QWidget" name="Log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>666</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>666</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>666</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>视唱练耳小助手</string>
  </property>
  <widget class="QPushButton" name="BackHome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>530</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
   <property name="styleSheet">
    <string notr="true">
     font: 12pt "方正小标宋简体";
     color: #433F67;
     background-color: #f3f3f3;
     border: 3px solid #808F9F;
     border-radius: 16px;
    </string>
   </property>
  </widget>
  <widget class="QLabel" name="loginDaysLabel">
   <property name="geometry">
    <rect>
     <x>1137</x>
     <y>300</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>25</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;方正小标宋简体&quot;;
color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>99</string>
   </property>
  </widget>
  <widget class="QChartView" name="listenTrendChartView" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>619</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QChartView" name="scoresTrendChartView" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>619</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QChartView" name="listenPieChartView" native="true">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>340</y>
     <width>311</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QChartView" name="scoresPieChartView" native="true">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>70</y>
     <width>311</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QtCharts/qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
